go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/e-breuninger/terraform-provider-netbox
 
 go 1.17
 
+replace github.com/fbreckle/go-netbox v0.0.0-20220117123939-7f4dc1816b81 => github.com/chapsuk/go-netbox v0.0.0-20220330084927-d140aa5aadf5
```

Comment:
This should be replaces with the upstream go-netbox from fbreckle, right? 
https://github.com/fbreckle/go-netbox/pull/11/files

Reply:
Yep